core-services/dashboard-analytics/src/main/java/com/tarento/analytics/handler/AdvanceTableChartResponseHandler.java

```diff
@@ -119,6 +119,19 @@ public AggregateDto translate(AggregateRequestDto requestDto, ObjectNode aggrega
             });
 
         });
+        if (chartNode.has(INCLUDE_CONSTANT_COLUMNS)) {
+            JsonNode constColumns = chartNode.get(INCLUDE_CONSTANT_COLUMNS);
+            Map<String, Plot> constPlotMap = new LinkedHashMap<>();
+            constColumns.forEach(item -> {
+                JsonNode value = aggregationNode.findValue(item.asText()).findValue("value");
+                Plot tempPlot = new Plot(item.asText(), value.asDouble(), "number");
+                constPlotMap.put(item.asText(), tempPlot);
+            });
+            for (String key : mappings.keySet()) {
+                Map<String, Plot> plotMap = mappings.get(key);
+                plotMap.putAll(constPlotMap);
```

Comment:
can this be moved out? why  do you need to run this in loop? This is anyways going to add keys only once right?